tests/pipelines/mochi/test_mochi.py

```diff
@@ -261,7 +264,10 @@ def test_vae_tiling(self, expected_diff_max: float = 0.2):
 
 
 @slow
+@nightly
```

Comment:
@sayakpaul Is this right to add here?

Reply:
Yes, but also test this by SSHing into the runner and running the test actually: https://github.com/huggingface/diffusers/blob/99c0483b67427de467f11aa35d54678fd36a7ea2/.github/workflows/nightly_tests.yml#L189

Reply:
Can remove the slow decorator